lib/data/external/GcpClient.js

```diff
@@ -91,6 +95,198 @@ class GcpClient extends AwsClient {
             return callback(null, gcpResp);
         });
     }
+
+    createMPU(key, metaHeaders, bucketName, websiteRedirectHeader, contentType,
+        cacheControl, contentDisposition, contentEncoding, log, callback) {
+        const metaHeadersTrimmed = {};
+        Object.keys(metaHeaders).forEach(header => {
+            if (header.startsWith('x-amz-meta-')) {
+                const headerKey = header.substring(11);
+                metaHeadersTrimmed[headerKey] = metaHeaders[header];
+            }
+        });
+        Object.assign(metaHeaders, metaHeadersTrimmed);
```

Comment:
Seems like we would not want to preserve the 'untrimmed' headers and store them with the 'trimmed' ones. Just want to confirm that this is intentional.

Reply:
this is definitely a bug. will fix this.

Reply:
this is definitely not intentional. will fix this

Reply:
Thanks, @alexanderchan-scality. 🚀